.travis.yml

```diff
@@ -16,7 +16,7 @@ install:
 - cp /usr/lib/python2.7/dist-packages/fontforge.* /home/travis/virtualenv/python2.7.13/lib/python2.7/site-packages
 - pip install -e .
 - pip install git+https://github.com/behdad/fonttools.git
-- pip install -U git+https://github.com/source-foundry/font-v.git@libfv
+- pip install font-v
```

Comment:
We don't need to explicitely install it here, because it is listed as a dependency on fonbakery's setup.py so it will be installed by ``pip install -e .``

Please delete the ``pip install font-v`` line.

Reply:
well, same goes for fonttools, fwiw..
Don’t count on master being stable. We reserve the right to change stuff before a new release is pushed

Reply:
done!